### PR TITLE
Update Snow Software entry

### PR DESCRIPTION
**Pull request by Alastair7999**
Thank you for your contribution! Some pointers to get it merged quickly:

For contributions in `software/`:

  - [x] PR Title: Please use "Add <vendor/product name>" (instead of "Update README.md")
  - [ ] Status: please select a value from the status table at the top
  - [ ] Version: Status Vulnerable / Workaround? -> List vulnerable versions.
             Status Fix?                     -> List fixed versions.
  - [x] Links: make sure you link to a public statement by the developer.
    Alternatively, include and link a file in the `software/vendor-statements/`
    directory
  - [x] Please mind the sorting

### CODE REVIEW

File: software/README.md [hunk 1]
```diff
@@ -3495,8 +3495,8 @@ The `Version` relates to the `Status` column. If `Status` field is set to 'Vulne
...
 | Sitecore    | Sitecore Managed Cloud | customers who do not use Solr |  | Not vuln |  | | | [source](https://support.sitecore.com/kb?id=kb_article_view&sysparm_article=KB1001391) |
 | Smartbear | SoapUI | <= 5.6.0 |  | Vulnerable |  | | | [source](https://community.smartbear.com/t5/SoapUI-Open-Source-Questions/Log4j-CVE-2021-44228-Zero-Day-Vulnerability-SOAP-UI-V5-6-0-or/m-p/227418) |
-| Snow Software | Snow Commander | 8.0.x - 8.9.x | Not vuln | Workaround |  | | | [source](https://community.snowsoftware.com/s/article/LOG4J-Vulnerability), [workaround](https://community.snowsoftware.com/s/article/SnowCommander-LOG4J2-XML-library-version-used-is-exposed-to-VULNERABILITY-CVE-2021-44228)|
-| Snow Software | VM Access Proxy | >= 3.0 | Not vuln | Workaround |  | | | [source](https://community.snowsoftware.com/s/article/LOG4J-Vulnerability), [workaround](https://community.snowsoftware.com/s/article/VM-ACCESS-PROXY-LOG4J2-XML-library-version-used-is-exposed-to-VULNERABILITY-CVE-2021-44228)|
+| Snow Software | Snow Commander | 8.0.x - 8.10.2 | Not vuln | Fix | Fix | Fix | Upgrade to 8.10.3 | [source](https://community.snowsoftware.com/s/article/LOG4J-Vulnerability) |
```
Comment (tschmidtb51) [inline software/README.md:3498]:
```suggestion
| Snow Software | Snow Commander | 8.10.3 | Not vuln | Fix | Fix | Fix | | [source](https://community.snowsoftware.com/s/article/LOG4J-Vulnerability) |

File: software/README.md [hunk 2]
```diff
@@ -3495,8 +3495,8 @@ The `Version` relates to the `Status` column. If `Status` field is set to 'Vulne
...
+| Snow Software | VM Access Proxy | 3.0 - 3.6 | Not vuln | Fix | Fix | Fix | Upgrade to 3.7 | [source](https://community.snowsoftware.com/s/article/LOG4J-Vulnerability) |
```
Comment (tschmidtb51) [inline software/README.md:3499]:
```suggestion
| Snow Software | VM Access Proxy | 3.7 | Not vuln | Fix | Fix | Fix |  | [source](https://community.snowsoftware.com/s/article/LOG4J-Vulnerability) |

**Review by tschmidtb51 — CHANGES_REQUESTED** [review]
hi @Alastair7999, thanks for contributing to the software list. Can you have a look at my comments? Some changes are necessary before we can merge.

**Review by AlastairPooley — APPROVED** [review]
Got it - if fixed, then just list fixed version

**Review by tschmidtb51 — CHANGES_REQUESTED** [review]
@Alastair7999 Thank you for your contribution.

Due to Github limits, we needed to reorganize the repo. Please close this PR and re-submit your changes.

**Pull request closed without merging**